### PR TITLE
lec01-02 notes

### PR DESCRIPTION
**Pull request by brendangubbins**
(attempting to) add lec01 and lec02 notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ My lectures can be listened to as youtube tracks and lecture notes are also scan
...
 * Lecture 2 video on slack
-  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec02kap.pdf)
+  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec02kap.pdf) [(Brendan Gubbins)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec02gubbins.pdf)
 * Lecture 1 video on slack
-  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec01kap.pdf) [(Elizabeth McHugh)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01mchugh.pdf)
+  - Theory Notes [(Prof)](https://github.com/kapelner/QC_Math_342W_Spring_2021/blob/master/lectures/lec01kap.pdf) [(Elizabeth McHugh)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01mchugh.pdf) [(Brendan Gubbins)](https://github.com/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01gubbins.pdf)
```
Comment (kapelner) [inline README.md]: You need to have "kapelner" there. This was my fault doing stuff late at night last night and I left it out of Elizabeth's pull request.

https://github.com/kaplener/QC_MATH_342W_Spring_2021/blob/master/lectures/lec01gubbins.pdf

Comment (brendangubbins) [reply README.md]: Thanks! Officially an open source contributor now

**Comment by kapelner** [comment]
You made it!

**Comment by kapelner** [comment]
I've changed the policy for these pull requests. You now keep the PDF file on your own forked repo under the lectures folder and only add a link in the README. Easier for everyone! 

See updated syllabus for example.

**Pull request merged**